lib/vsc/utils/fancylogger.py

```diff
@@ -450,18 +450,24 @@ def logToFile(filename, enable=True, filehandler=None, name=None, max_bytes=MAX_
 
     if you want to disable logging to file, pass the earlier obtained filehandler
     """
-    handleropts = {'filename': filename,
-                   'mode': 'a',
-                   'maxBytes': max_bytes,
-                   'backupCount': backup_count,
-                   }
-    return _logToSomething(logging.handlers.RotatingFileHandler,
-                           handleropts,
-                           loggeroption='logtofile_%s' % filename,
-                           name=name,
-                           enable=enable,
-                           handler=filehandler,
-                           )
+    handleropts = {
+        'filename': filename,
+        'mode': 'a',
+        'maxBytes': max_bytes,
+        'backupCount': backup_count,
+    }
+    # logging to a file is going to create the file later on, so let's try to be helpfull and create the path if needed
+    directory = os.path.dirname(filename)
+    if not os.path.exists(directory):
+        os.makedirs(directory)
```

Comment:
what? I'm not following your argument here? Is this meant to stdweird's remarks, or as a remark to my latest remark?

I agree we shouldn't handle exceptions here, just let them flow...

And certainly not generate new exceptions, I completely agree there.

a traceback is the best way to later on figure out where exactly things broke, it is the second most usefull thing you could have, after the exceptions type.
I agree, creating the directory failed by a problem that's out of control of fancylogger, so why should it get in between there and claim to know something more then it does, in the current implementation it's not trying to do any of that, so please feel free to tell me this is correct ;-)

The newly proposed `raise` to come forward to stdweirds remarks would also not produce a traceback, just preserve the original one, and make sure the raised exception is left mostly intact with only an extra (potentially helpful?) message.